includes/class-wc-payments-onboarding-service.php

```diff
@@ -1024,4 +1031,66 @@ public function maybe_add_test_drive_settings_to_new_account_request( array $arg
 
 		return $args;
 	}
+
+	/**
+	 * Updates the enabled payment methods for a gateway.
+	 *
+	 * @param WC_Payment_Gateway_WCPay $gateway Payment gateway instance.
+	 * @param array                    $capabilities Provided capabilities.
+	 */
+	public function update_enabled_payment_methods_ids( $gateway, $capabilities = [] ): void {
+		$enabled_gateways = $gateway->get_upe_enabled_payment_method_ids();
+
+		$enabled_payment_methods = array_unique(
+			array_merge(
+				$enabled_gateways,
+				$this->exclude_placeholder_payment_methods( $capabilities )
+			)
+		);
+
+		// Update the gateway option.
+		$gateway->update_option( 'upe_enabled_payment_method_ids', $enabled_payment_methods );
+
+		foreach ( $enabled_payment_methods as $payment_method_id ) {
+			$payment_gateway = WC_Payments::get_payment_gateway_by_id( $payment_method_id );
+			if ( $payment_gateway ) {
+				$payment_gateway->enable();
+				$payment_gateway->update_option( 'upe_enabled_payment_method_ids', $enabled_payment_methods );
+			}
+		}
+
+		// If WooPay is enabled, update the gateway option.
+		if ( isset( $capabilities['woopay'] ) && $capabilities['woopay'] ) {
```

Comment:
You could just check for `! empty( $capabilities['woopay'] )`

Reply:
No. `empty()` also checks for existence ([docs](https://www.php.net/manual/en/function.empty.php)).

Reply:
TIL! Thank you, updated!